app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -254,10 +255,7 @@ class MainActivity : ComponentActivity() {
                     contentColor = MaterialTheme.colors.onPrimary,
                     modifier = Modifier.size(80.dp)
                 ) {
-                    Text(
-                        text = "+",
-                        fontSize = 52.sp
-                    )
+                    Icon(painterResource(id = R.drawable.ic_add_foreground), "add")
```

Comment:
Instead of hardcoding a text label for a button every time you have an add button you can either create a new composable class called 'AddButton', or reference an Icon. I chose to reference an Icon because you may have different versions of the add button for different scenarios. You can also tweak the icon to change it's color, size, image, and other attributes. https://developer.android.com/studio/write/create-app-icons